apps/synapse-portal-framework/src/components/elportal/ELContributeYourData.tsx

```diff
@@ -0,0 +1,97 @@
+import { Box, Link, Button } from '@mui/material'
+import React from 'react'
+import backgroundSpotsLeft from './assets/dot_blob_top_left.png'
+import backgroundSpotsRight from './assets/dot_blob_bottom_right.png'
+
+const ELContributeYourData: React.FC = () => {
+  return (
+    <Box
+      sx={{
+        backgroundColor: '#5BA998',
+        height: '250px',
+        textAlign: 'left',
+        display: 'flex',
+        alignItems: 'center',
+        position: 'relative',
+        overflow: 'hidden',
+      }}
+    >
+      <Box
+        sx={{
+          flex: '0 0 auto',
+          width: '33%',
+          height: '100%',
+          backgroundImage: `url(${backgroundSpotsLeft})`,
+          backgroundRepeat: 'no-repeat',
+          backgroundPosition: 'top left',
+          backgroundSize: '100%',
+          zIndex: 0,
+          overflow: 'clip',
+        }}
+      />
+      <Box
+        sx={{
+          flex: '1 1 auto',
+          zIndex: 1,
+          textAlign: 'center',
+          paddingTop: '100px',
+          paddingBottom: '100px',
+        }}
+      >
+        <Box
+          sx={{
+            width: '90%',
+            height: '3px',
+            backgroundColor: 'white',
+            opacity: 0.5,
+            marginBottom: 2,
+            margin: '0 auto',
+          }}
+        />
```

Comment:
Since this horizontal line seems to be the same width as the headline text, I would suggest making it a `borderTop` property on the headline component, maybe adding these `sx` properties to the `Typography` component would work?

```
            pt: 2,
            mx: 'auto',
            width: 'max-content',
            borderTop: '6px solid #ffffff88',
```

Reply:
thank you! That was bugging me, I couldn't get the bar to work fit `fit-content`. I'll give this a shot.